src/components/toast/toast.stories.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+
+
+import readme from './readme.md';
+
+export default {
+  title: 'Toast',
+  decorators: [withKnobs],
+  parameters: {
+    notes: { markdown: readme },
+  }
+};
+
+export const toastSystem = () => {
+  const variantOptions = { System: 'system', Error: 'error', Success: 'success', Warning: 'warning' };
+  const actionTypeOptions = { Button: 'button', Icon: 'icon' };
+  const buttonActionOptions = { Close: 'close', Custom: 'custom' };
+
```

Comment:
Why are those Options necessary? Can't we the types, e. g. `VariantType`?

Reply:
this is just a shortcut to let the code of the story more clear